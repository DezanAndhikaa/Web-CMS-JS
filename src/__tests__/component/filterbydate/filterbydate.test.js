import React from 'react';
import Enzyme, { shallow } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import FilterByPeriodDate from '../../../components/FilterByPeriodeDate/FilterByPeriodeDate';

Enzyme.configure({adapter:new Adapter()});
it("render filter by period date correctly", () => {
    const wrapper = shallow (
        <FilterByPeriodDate />
    )
    expect(wrapper).toMatchSnapshot();
})