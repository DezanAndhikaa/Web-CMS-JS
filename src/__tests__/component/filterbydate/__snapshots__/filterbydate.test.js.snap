// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render filter by period date correctly 1`] = `ShallowWrapper {}`;
